docs/pt/docs/advanced/response-directly.md

```diff
@@ -0,0 +1,70 @@
+# Retornando uma Resposta Diretamente
+
+Quando você cria uma *operação de rota* no **FastAPI** você pode retornar qualquer dado nela: um dicionário (`dict`), uma lista (`list`), um modelo do Pydantic ou banco de dados, etc.
+
+Por padrão, o **FastAPI** irá converter automaticamente o valor do retorno para JSON utilizando o `jsonable_encoder` explicado em [JSON Compatible Encoder](../tutorial/encoder.md){.internal-link target=_blank}.
+
+Então, por baixo dos panos, ele incluiria esses dados compatíveis com JSON (e.g. um `dict`) dentro de uma `JSONResponse` que é utilizada para enviar uma resposta para o cliente.
+
+Mas você pode retornar a `JSONResponse` diretamente nas suas *operações de rota*.
+
+Pode ser útil para retornar cabeçalhos e cookies personalizados, por exemplo.
+
+## Retornando uma `Response`
+
+Na verdade, você pode retornar qualquer `Response` ou subclasse dela.
+
+/// tip | Dica
```

Comment:
I really don't know if the call-outs need or don't need double quotes. But all the translations that I see uses the double quotes for the translated part, as you can see in:

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/environment-variables.md?plain=1#L3

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/virtual-environments.md?plain=1#L5

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/python-types.md?plain=1#L15

and etc...

Reply:
I searched here, and apparently the docs used to use the version with !!! Tip "Dica". Back then, it used the double quotes. But now we don't need it anymore.

Just ignore this reviewed line

Reply:
I also found it weird when i saw the double quotes being rendered in the page, maybe it's an update on mkdocs.